src/js/health-records/containers/DownloadPage.jsx

```diff
@@ -102,18 +104,44 @@ export class DownloadPage extends React.Component {
     return <DownloadLink ref={e => { this.pdfDownloadButton = e; }} onClick={linkOnClick} name="PDF File" docType="pdf"/>;
   }
 
+  renderRequestDate() {
+    const { form } = this.props;
+
+    if (!form.requestDate) {
```

Comment:
Nitpick: Noticed this is split into multiple lines while a similar check below is all in one.